src/BookNavigator/downloads/downloads.js

```diff
@@ -59,12 +69,24 @@ export class IABookDownloads extends LitElement {
     `;
   }
 
+  get installSimplyEAldikoThoriumMsg() {
+    return html`
+    <p>For LCP downloads, make sure you have SimplyE or Aldiko Next installed on mobile or Thorium on desktop.</p>
+    <ul>
+      <li><a href="https://librarysimplified.org/simplye/" rel="noopener noreferrer" target="_blank">Install SimplyE</a></li>
+      <li><a href="https://www.demarque.com/en-aldiko" rel="noopener noreferrer" target="_blank">Install Aldiko</a></li>
+      <li><a href="https://www.edrlab.org/software/thorium-reader/" rel="noopener noreferrer" target="_blank">Install Thorium</a></li>
+    </ul>
+  `;
+  }
+
   render() {
     return html`
       ${this.header}
       ${this.loanExpiryMessage}
       <ul>${this.renderDownloadOptions()}</ul>
-      ${this.isBookProtected ? this.accessProtectedBook : nothing}
+      ${this.hasLCPOption ? this.installSimplyEAldikoThoriumMsg : nothing}
+      ${this.isBookProtected && !this.hasLCPOption ? this.accessProtectedBook : nothing}
```

Comment:
TINY SUGGESTION: I think you could slightly simplify the logic by nesting the conditions, e.g.
```
${this.hasLCPOption
  ? this.installSimplyEAldikoThoriumMsg
  : (this.isBookProtected ? this.accessProtectedBook : nothing)
}
```
Not sure it's an improvement, but see what you think! ;)

Reply:
hi, very good suggestion! updated.